io.openems.edge.evcs.keba.kecontact/src/io/openems/edge/evcs/keba/kecontact/EvcsKebaKeContactImpl.java

```diff
@@ -310,26 +309,9 @@ private int calculateCurrent(int power, int phases) {
 		current = Math.min(current, 63_000);
 
 		if (current < 6000) {
-			current = 0;
+			current = 6000; // Set to minimum allowed value
```

Comment:
Das hattest du inital auch in dem Pull Request.
Wenn gerade weniger als 6 A verfügbar sind, können wir nicht einfach eine Ladevorgabe erlauben.

Reply:
Achso, ich dachte das wäre nach wie vor ein Fehler - dann sorry..